src/functions/durableOrchestration.ts

```diff
@@ -0,0 +1,36 @@
+import { app, HttpHandler, HttpRequest, HttpResponse, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { ActivityHandler, OrchestrationHandler } from 'durable-functions';
+
+// Replace with the name of your Durable Functions Activity
+const activityName = 'hello';
+
+const orchestrator: OrchestrationHandler = function* (context) {
+    const outputs = [];
+    outputs.push(yield context.df.callActivity(activityName, 'Tokyo'));
+    outputs.push(yield context.df.callActivity(activityName, 'Seattle'));
+    outputs.push(yield context.df.callActivity(activityName, 'Cairo'));
+
+    return outputs;
+};
+df.orchestration('durableOrchestrator1', orchestrator);
+
+const helloActivity: ActivityHandler<string> = (_context: InvocationContext, input: string) => {
```

Comment:
> For example, if you could somehow use generics to link the types for input on the ActivityHandler and input when calling context.df.callActivity, I think you could prevent a bug where you pass an object into an activity that expects a string.

Can't we link them by also making `callActivity()` a generic and saying that `callActivity<T>()` can only be passed an `ActivitiyHandler<T>`? @davidmrdavid and I talked already about the possibility of making `callActivity()` take in some kind of object or handler rather than just a string name

Reply:
Maybe, but I think it might feel weird to pass the handler into callActivity - we're not using that pattern anywhere else. I would have to see/test your changes, and the much safer/faster thing is to just remove generics for this PR

Reply:
I really like @hossam-nasr's idea, but I agree that it isn't necessary for this first minimal version.
With this new programming model, we have opportunities for type safety that we want to explore, but for the time being I also think it's best not to commit to them.

Reply:
Perhaps we can explore this further in a separate PR, but I've already removed the generics for the purposes of this PR